nextflow.yaml

```diff
@@ -0,0 +1,65 @@
+package:
+  name: nextflow
+  version: 24.10.3
+  epoch: 0
+  description: A DSL for data-driven computational pipelines.
+  copyright:
+    - license: Apache-2.0
+  dependencies:
+    runtime:
+      - openjdk-21
```

Comment:
do we know if we need full java here? We can consider only adding jre at runtime.